### DISCUSSION TITLE
Error en la ejecución de test e2e en mi local

### DISCUSSION THREAD

**Discussion by JuaniLezcano** [frlp-utn-ingsoft/catedra#153] [Q&A] [answered]
Buenas tardes, quería consultar si saben porque me ocurre este error. Estoy trabajando en mi local con una MV con Ubuntu, dias atras me funcionaba perfecto la ejecución de python manage.py test functional_tests pero de la nada se me rompio.
Igualmente los chequeos se realizan en la CI por eso me quedo tranquilo, pero me gustaria poder resolver el problema.
![image](https://github.com/frlp-utn-ingsoft/catedra/assets/107371324/a5ddf0cd-a8a9-4fb1-a8e5-3f60b0217216)
![image](https://github.com/frlp-utn-ingsoft/catedra/assets/107371324/1c55acb4-b5f7-4938-a8bd-71680f1f169a)

**Comment by RodrigoJacznik** [comment]
Hola, me pasas el link a tu repo y los pruebo para ver si lo puedo reproducir?

**Reply by JuaniLezcano** [reply]
Te envío el link. https://github.com/JuaniLezcano/vetsoft , igualmente al resto de los chicos del grupo les funciona joya, de hecho a mi en la notebook también. Debo tener algún problema con el entorno dentro de la MV pero no se si es del playwright o alguna otra dependencia, me volví loco buscando y preguntándole al chat gpt jaja. Si es muy complicado el asunto no pasa nada.

**Reply by RodrigoJacznik** [reply, marked as answer]
Lo probaste corriendo no en forma headless? `HEADLESS=0 python manage.py test functional_tests`. El browser arbe?

**Reply by JuaniLezcano** [reply]
El browser abre pero no la app, se me queda con el browser en blanco y me tira el error. Igual Rodri no te hagas drama, me hago otra MV e instalo todo de nuevo. Mil gracias!
